system/jhtree/ctfile.cpp

```diff
@@ -1126,6 +1106,27 @@ CJHTreeBlobNode::~CJHTreeBlobNode()
 {
 }
 
+void CJHTreeBlobNode::unpack(const void *nodeData, bool needCopy)
+{
+    CJHTreeNode::unpack(nodeData, needCopy);
+    const byte *data = ((const byte *) nodeData) + sizeof(hdr);
+    char keyType = keyHdr->getKeyType();
+    if (keyType & HTREE_COMPRESSED_KEY)
+    {
+        expandedSize = keyHdr->getNodeSize();
+        keyBuf = expandKeys(data, expandedSize);
+    }
+    else
+    {
+        // Do blobs work on indexes that are not compressed - looking at old code I'm not sure they do
+        assertex(!(keyType & COL_PREFIX));
+        expandedSize = hdr.keyBytes + sizeof( __int64 );  // MORE - why is the +sizeof() there?
+        keyBuf = (char *) allocMem(expandedSize);
+        memcpy(keyBuf, data, hdr.keyBytes + sizeof( __int64 ));
+
```

Comment:
trivial/formatting: extra newline

Reply:
This code has been deleted in a later PR